OMNet.cmake

```diff
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.20)
```

Comment:
I am not happy with requiring 3.20 in general. With respect to Artery, my goal is to support the current Debian stable. At the moment Debian stable ships with 3.13.
Can we please make this a Windows-only requirement?

Reply:
Yes... I get your point, modern CMake gets more and more really useful features...
I guess there is some ugly workaround for the feature we need, let me investigate...

Reply:
It seems that CMake `file` module offers this feature for ages. I will change that syntax and set the version to 3.13.